cpp/src/Glacier2CryptPermissionsVerifier/Makefile.mk

```diff
@@ -6,7 +6,7 @@ $(project)_libraries                            += Glacier2CryptPermissionsVerif
 
 Glacier2CryptPermissionsVerifier_targetdir        := $(libdir)
 Glacier2CryptPermissionsVerifier_dependencies     := Glacier2 Ice
-Glacier2CryptPermissionsVerifier[shared]_cppflags := -DCRYPT_PERMISSIONS_VERIFIER_API_EXPORTS
+Glacier2CryptPermissionsVerifier_cppflags         := -DCRYPT_PERMISSIONS_VERIFIER_API_EXPORTS
```

Comment:
Reverted an earlier commit that add "[shared]". We want to use exactly the same flags for static and shared builds on macos.